spyrmsd/molecule.py

```diff
@@ -155,11 +202,11 @@ def to_graph(self):
         The molecular graph is cached.
         """
         if self.G is None:
-            if self.adjacency_matrix is not None:
+            try:
```

Comment:
`self.adjacency_matrix` does not exist if `adjacency_matrix=None` when creating a `molecule.Molecule`. Use `try/except` instead to catch missing attribute.

This solves `mypy` problems in `test_graph_from_atomic_coordinates_perception`:
```diff
-    m.adjacency_matrix = None
+    delattr(m, "adjacency_matrix")
```